tests/driver_io1_xplained/main.c

```diff
@@ -21,16 +21,37 @@
 #include <stdio.h>
 #include <inttypes.h>
 
-#include "io1_xplained.h"
-#include "io1_xplained_params.h"
 #include "xtimer.h"
 #include "board.h"
 
-#define SLEEP_1S   (1 * 1000 * 1000u) /* 1 seconds delay between each test */
+#include "periph/gpio.h"
+
+#include "at30tse75x.h"
+#include "sdcard_spi.h"
+#include "io1_xplained.h"
+#include "io1_xplained_params.h"
+
+#define DELAY_1S   (1U) /* 1 seconds delay between each test */
```

Comment:
Why this rename?

Reply:
> Why this rename?

The comment is stating a delay, so I made it more in line with the comment